apps/web/components/molecules/opinions/opinions.mock.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Opinion } from '../../../types/types';
+
+export const opinions: Opinion[] = [
+	{
+		author: 'Jan Kowalski',
+		avatar:
+			'https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2960&q=80',
+		rating: 5,
+		comment:
+			'Zadanie programistyczne, które otrzymałem, było dobrze sformułowane i precyzyjnie opisywało wymagania dotyczące tworzenia aplikacji internetowej. Zadanie miało na celu zaimplementowanie funkcjonalności wyszukiwania produktów w bazie danych i wyświetlenia wyników na stronie internetowej.',
+	},
+	{
+		author: 'Jan Kowalski',
+		avatar:
+			'https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2960&q=80',
+		rating: 5,
+		comment:
+			'Zadanie programistyczne, które otrzymałem, było dobrze sformułowane i precyzyjnie opisywało wymagania dotyczące tworzenia aplikacji internetowej. Zadanie miało na celu zaimplementowanie funkcjonalności wyszukiwania produktów w bazie danych i wyświetlenia wyników na stronie internetowej.',
+	},
+	{
+		author: 'Jan Kowalski',
+		avatar:
+			'https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2960&q=80',
+		rating: 4,
+		comment:
+			'Zadanie programistyczne, które otrzymałem, było dobrze sformułowane i precyzyjnie opisywało wymagania dotyczące tworzenia aplikacji internetowej. Zadanie miało na celu zaimplementowanie funkcjonalności wyszukiwania produktów w bazie danych i wyświetlenia wyników na stronie internetowej.',
```

Comment:
I know this is mock data but it would be cool to see some variety and how it looks and behaves with different lengths of the comments. 😄 
Also is it possible for the rating to be `0` or `negative` or above `5`? What then?

Reply:
User can't give negative or above 5 rating

Reply:
Is there any validation for that on the backend?

Reply:
I think not now